dotnetflix.Web/Pages/Manager/ManagerPage.razor

```diff
@@ -1,4 +1,5 @@
 @page "/manager"
+@attribute [Authorize(Roles = "Manager")]
```

Comment:
Deze moet ook toegevoegd worden aan alle manager pages, zoals https://localhost:7257/manager/seats. Maar is geen prio voor demo van Auth0